Lib/test/test_ssl.py

```diff
@@ -2471,6 +2470,39 @@ def test_check_hostname(self):
                                                 "check_hostname requires server_hostname"):
                         context.wrap_socket(s)
 
+        def test_check_hostname_idn(self):
+            if support.verbose:
+                sys.stdout.write("\n")
+
+            server_context = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
```

Comment:
I don't know, I just made it match all the other tests around it (aka this is blatantly copy/paste code).